src/main/java/com/zufar/icedlatte/security/endpoint/UserSecurityEndpoint.java

```diff
@@ -118,19 +112,10 @@ public ResponseEntity<Void> forgotPassword(@RequestBody final ForgotPasswordRequ
     @PostMapping("/password/change")
     public ResponseEntity<Void> changePassword(@RequestBody final ChangePasswordRequest request) {
         log.info("Received change password request for user");
-        UserDto userDto = singleUserProvider.getUserByEmail(request.email());
-        emailTokenConformer.confirmResetPasswordEmailByCode(
-                new com.zufar.icedlatte.security.dto.ConfirmEmailRequest(request.code()));
-        changeUserPasswordOperationPerformer.changeUserPassword(userDto.getId(), request.password());
+        UserDto userDto = singleUserProvider.getUserByEmail(request.getEmail());
+        emailTokenConformer.confirmResetPasswordEmailByCode(new ConfirmEmailRequest(request.getCode()));
+        changeUserPasswordOperationPerformer.changeUserPassword(userDto.getId(), request.getPassword());
         log.info("Password changed for user");
         return ResponseEntity.ok().build();
     }
-
-    // for testing only
-    @PostMapping("/email/code")
-    public ResponseEntity<String> getEmailVerificationCode(@RequestBody final UserRegistrationRequest request) {
-        String token = emailTokenSender.getEmailVerificationCode(request);
-        return ResponseEntity.ok()
-                .body(token);
-    }
 }
```

Comment:
I'm afraid this method was requested by QA , we should probably discuss it with Tanya